x/auth/ante/setup_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func TestSetupDecorator_BlockMaxGas(t *testing.T) {
-	suite := SetupTestSuite(t, true)
+	suite := SetupTestSuite(t, false)
```

Comment:
In here and the other following tests: Why are we setting `isCheckTx` to false ?

Reply:
same as https://github.com/0xPolygon/cosmos-sdk/pull/3#discussion_r1502495753